FAO Publications.js

```diff
@@ -153,7 +158,7 @@ function scrape(doc, url) {
 		if (!subTitle) {
 			newItem.title = mainTitle;
 		}
-		else if ((newItem.language == 'zh') || (newItem.language == 'ja') || (newItem.language == 'ko')) {
+		else if ((newItem.language == 'zh') || (newItem.language == 'ja')) {
```

Comment:
Why exclude Korean? Doesn't seem relevant to DOI changes.

Reply:
No this is not related to DOI changes. I confirmed that Korean uses half-width punctuations (":"), not full-width ones like Chinese and Japanese ("："), so I removed it.